extensions/wikia/PotentialMemberPageExperiments/scripts/entry-point-experiment.js

```diff
@@ -74,15 +87,26 @@ require([
 	}
 
 	function experimentEnabled() {
+		experimentGroup = abTest.getGroup(experimentNames[contentLanguage]);
 		return experiments.hasOwnProperty(experimentGroup);
 	}
 
 	function setupExperiment(resources) {
-		var experiment = experiments[experimentGroup];
+		var experiment = experiments[experimentGroup],
+			templateData;
 
 		loader.processStyle(resources.styles);
+		mw.messages.set(resources.messages);
+
+		templateData = {
+			bannerType: experiment.type,
+			language: contentLanguage,
+			messageText: mw.message('pmp-entry-point-message').plain(),
```

Comment:
@Grunny - I wasn't sure about the usage of `plain()`and `{{{messageText}}}` here. Is there a better way to deal with messages including HTML tags? Thanks